scripts/deploy.mjs

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env zx
+
+import { $, which } from 'zx'
+import prompts from 'prompts'
+import semver from 'semver'
+import { readFileSync } from 'fs'
+import { bold, green, red } from 'kleur/colors'
+
+const print = (...args) => {
+  // eslint-disable-next-line no-undef, no-console
+  console.log(...args)
+}
+
+const exit = (code) => {
+  // eslint-disable-next-line no-undef
+  process.exit(code)
+}
+
+const PACKAGE_FOLDERS = [
+  {
+    title: 'Library',
+    value: 'field-plugin',
+  },
+  {
+    title: 'CLI',
+    value: 'cli',
+  },
+]
+
+// Check if `gh` exists
```

Comment:
Here I've assumed that you have installed GitHub CLI (`gh`):
https://cli.github.com/

It helps create a pull-request easily (it comes a bit later)

Reply:
I think its fine this way, you are even letting the user know to install it in case they dont have the dependency. Potentially we can add the dependency to the project? What do do you think?

Reply:
Unfortunately this is not an npm library. You need to install it in a different way (`brew install gh`)